File: app/src/main/java/ru/coffeeplanter/androidlogin/presentation/fragments/login/LoginPresenter.java
package ru.coffeeplanter.androidlogin.presentation.fragments.login;

import ru.coffeeplanter.androidlogin.presentation.BasePresenter;

interface LoginPresenter extends BasePresenter {

    void validateLoginAndPassword(String login, String password);
    void tryToLogin(String login, String password);
    void onFragmentPause();
    void onFragmentDestroy();

}

File: app/src/main/java/ru/coffeeplanter/androidlogin/presentation/fragments/login/LoginPresenter.kt
package ru.coffeeplanter.androidlogin.presentation.fragments.login

import ru.coffeeplanter.androidlogin.presentation.BasePresenter

internal interface LoginPresenter : BasePresenter {

    fun validateLoginAndPassword(login: String, password: String)
    fun tryToLogin(login: String, password: String)
    fun onFragmentPause()
    fun onFragmentDestroy()

}
